src/GitHub.UI.Reactive/Controls/Validation/UserErrorMessages.cs

```diff
@@ -106,18 +106,14 @@ public UserError UserError
             Justification = "We're registering a handler for a type so this is appropriate.")]
         public IDisposable RegisterHandler<TUserError>(IObservable<bool> clearWhen) where TUserError : UserError
         {
-            if (IsVisible)
+            return UserError.RegisterHandler<TUserError>(userError =>
```

Comment:
The handler in the login dialog wasn't getting registered because the `UserErrorMessages` control wasn't visible at the time of the call. I saw that check and it looked wrong: if someone wants to not register a handler when a control isn't visible they should do it explicitly IMO. Doing nothing in a situation like this just seems confusing to me...